extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.tracking.js

```diff
@@ -0,0 +1,35 @@
+define('wikia.articleVideo.featuredVideo.tracking', [], function () {
+	function getDefaultState() {
+		return {
+			wasAlreadyUnmuted: false
+		}
+	}
+
+	function track(gaData) {
+		console.log(gaData);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)